rocky/reports/views/generate_report.py

```diff
@@ -175,7 +175,20 @@ def generate_reports_for_oois(self) -> dict[str, dict[str, dict[str, Any]]]:
                 continue
 
             for ooi, data in results.items():
-                ooi_human_readable = Reference.from_str(ooi).human_readable
+                tokenized_ooi = Reference.from_str(ooi).tokenized
```

Comment:
Duplicate references to `Reference.from_str(ooi)`, can be extracted to a variable:
`reference = Reference.from_str(ooi)`